regulation-worker/internal/delete/api/api.go

```diff
@@ -81,7 +81,7 @@ func (api *APIManager) deleteWithRetry(ctx context.Context, job model.Job, desti
 	}
 
 	defer stats.Default.NewTaggedStat(
-		"file_cleaning_time",
+		"regulation_worker_file_cleaning_time",
```

Comment:
The name of this measurement is weird. What file is it referring to?

Reply:
for `batch` destination, it refers to the files that we clean during the deletion process. But, I believe this name doesn't makes sense in case of `api` & `kvstore` destination. 
Do you think `regulation_worker_cleaning_time` makes sense? with `destType` having module name?